connectors/connector-revert/server.ts

```diff
@@ -23,36 +23,39 @@ export const revertServer = {
           continue
         }
         const sState = ((state ?? {}) as Record<string, unknown>)[name] ?? {}
-        yield* iterateRecordsInStream(name, stream.fields ?? [], sState) // TODO(@jatin): update this.
+        yield* PipeRecordsInStream(name, stream.fields ?? [], sState) // TODO(@jatin): update this.
       }
     }
 
-    async function* iterateRecordsInStream(
+    async function* PipeRecordsInStream(
       stream: string,
       /** stream state */
       fields: string[],
       sState: {cursor?: string | null},
     ) {
       const plural = revertPluralize(stream)
       let cursor = sState.cursor
+      const records = []
       while (true) {
         // @jatinsandilya don't worry about making this work for incremental sync
         // Our requirement so far is just one time import for now
         const res = await instance.GET(`/crm/${plural as 'companies'}`, {
           params: {query: {cursor, fields: fields.join(',')}},
         })
-        yield res.data.results.map((com) =>
-          helpers._opData(stream as 'company', com.id ?? '', com),
-        )
+
+        records.push(...res.data.results)
         cursor = res.data.next
         if (!cursor) {
+          yield helpers._opData(stream as 'company', '', {
+            [stream]: records,
```

Comment:
This doesn't quite look right. We need to emit individual records, see example in QBO/server